### PR TITLE
chore: Add personnal information page

### PR DESCRIPTION
**Pull request by Wandarry**
### PR children

- [x] PR #43 
- [x] PR #44 
- [x] PR #45 
- [x] PR #46 

Add personnal information page pledging step

https://github.com/Wandarry/VitaNova/assets/101728187/a8ca16a6-f63d-4acd-b450-2607ea96a60b

### CODE REVIEW

**Comment by Wandarry** [comment]
Les radio buttons ne fonctionnent pas encore. De même que la vérification au niveau de NPI.

File: app/(tabs)/personalInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  completeName: "",
+  personalNumberIdentification: "",
+  phoneNumber: "",
+  email: "",
+  address: "",
+  gender: "",
+};
+
+export default function PersonalInformation() {
+  const [isLoading, setIsLoading] = useState(false);
```
Comment (BOCOVO) [inline app/(tabs)/personalInformation.tsx]:
```suggestion
  const [isSumitting, setIsSumitying] = useState(false);
```

File: app/(tabs)/personalInformation.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  const onSubmit = () => {
+    setIsLoading(true);
```
Comment (BOCOVO) [inline app/(tabs)/personalInformation.tsx]: `setIsLoading(false)`  is not missing?

File: app/(tabs)/personalInformation.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                    borderColor="$primaryLight"
+                  >
+                    <Text fontSize={15}>
+                      Veuillez indiquer votre nom complet, tel qu'il figure sur
+                      votre pièce d'identité.
+                    </Text>
+                    <TextInput
+                      placeholder="Nom complet"
+                      icon={User}
+                      name="completeName"
```
Comment (BOCOVO) [inline app/(tabs)/personalInformation.tsx]:
```suggestion
                      name="fullName"
```

File: app/(tabs)/personalInformation.tsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                    borderColor="$primaryLight"
+                  >
+                    <Box py="$8" gap="$4" w="50%">
+                      <Text fontSize={15}>
+                        Veuillez indiquer votre Numéro d'Identification
+                        Personnel
+                      </Text>
+                      <TextInput
+                        placeholder="NPI"
+                        icon={User}
```
Comment (BOCOVO) [inline app/(tabs)/personalInformation.tsx:86]: User icon ?

File: app/(tabs)/personalInformation.tsx [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                    borderBottomWidth={1}
+                    borderColor="$primaryLight"
+                  >
+                    <Text fontSize={15}>Veuillez indiquer votre sexe</Text>
+                    <HStack justifyContent="space-between">
+                      <RadioButton
+                        name="gender"
+                        label="Femme"
+                        value="female"
+                        isDisabled={false}
```
Comment (BOCOVO) [inline app/(tabs)/personalInformation.tsx:138]: isDisabled must be set to false as default.

File: app/(tabs)/personalInformation.tsx (hunk 4 above)
Comment (Wandarry) [reply app/(tabs)/personalInformation.tsx:86]: Yes

File: app/(tabs)/personalInformation.tsx (hunk 2 above)
Comment (Wandarry) [reply app/(tabs)/personalInformation.tsx]: Oh, yes, but first, shouldn't it provide that list of data to firestore ? This is just there for now...

File: app/(tabs)/personalInformation.tsx [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+                      />
+                      <RadioButton
+                        name="gender"
+                        label="Homme"
+                        value="female"
```
Comment (BOCOVO) [inline app/(tabs)/personalInformation.tsx]:
```suggestion
                        value="male"
```

**Pull request merged**